### DISCUSSION TITLE
Missing the P1 "Total Gas" entity

### DISCUSSION THREAD

**Discussion by VmaxTurbo** [fsaris/home-assistant-zonneplan-one#96] [Ideas]
This is a value in the P1 reading, is this value missing on the Zonneplan API?

I would like to have this for my Home Assistant energy dashboard, 
Now I'm using an additional P1 adapter from Homewizard, only for this value...

**Comment by fsaris** [comment]
That value isn't part of the API. At least it wasn't in the past.

Only values that are visible in the app are available
